api/tests/unit/infrastructure/serializers/jsonapi/certification-candidate-serializer_test.js

```diff
@@ -6,46 +6,71 @@ const _ = require('lodash');
-          'result-recipient-email': certificationCandidate.resultRecipientEmail,
-          'birthdate': certificationCandidate.birthdate,
-          'external-id': certificationCandidate.externalId,
-          'extra-time-percentage': certificationCandidate.extraTimePercentage,
-          'is-linked': !_.isNil(certificationCandidate.userId),
-        },
-      },
-    };
+    certificationCandidate = domainBuilder.buildCertificationCandidate({
+      schoolingRegistrationId: 1,
+    });
   });
 
   describe('#serialize()', () => {
 
     it('should convert a CertificationCandidate model object into JSON API data', function() {
+      // given
+      const expectedJsonApiData = {
+        data: {
+          type: 'certification-candidates',
+          id: certificationCandidate.id.toString(),
+          attributes: {
+            'first-name': certificationCandidate.firstName,
+            'last-name': certificationCandidate.lastName,
+            'birth-city': certificationCandidate.birthCity,
+            'birth-province-code': certificationCandidate.birthProvinceCode,
+            'birth-country': certificationCandidate.birthCountry,
+            'email': certificationCandidate.email,
+            'result-recipient-email': certificationCandidate.resultRecipientEmail,
+            'birthdate': certificationCandidate.birthdate,
+            'external-id': certificationCandidate.externalId,
+            'extra-time-percentage': certificationCandidate.extraTimePercentage,
+            'is-linked': !_.isNil(certificationCandidate.userId),
+            'schooling-registration-id': certificationCandidate.schoolingRegistrationId,
+          },
+        },
+      };
+
       // when
       const jsonApi = serializer.serialize(certificationCandidate);
 
       // then
-      expect(jsonApi).to.deep.equal(jsonApiData);
+      expect(jsonApi).to.deep.equal(expectedJsonApiData);
     });
 
   });
 
   describe('#deserialize()', () => {
 
     it('should convert JSON API data into a CertificationCandidate model object', async function() {
+      // given
+      const jsonApiData = {
+        data: {
+          type: 'certification-candidates',
+          id: certificationCandidate.id.toString(),
+          attributes: {
+            'first-name': certificationCandidate.firstName,
+            'last-name': certificationCandidate.lastName,
+            'birth-city': certificationCandidate.birthCity,
+            'birth-province-code': certificationCandidate.birthProvinceCode,
+            'birth-country': certificationCandidate.birthCountry,
+            'email': certificationCandidate.email,
+            'result-recipient-email': certificationCandidate.resultRecipientEmail,
+            'birthdate': certificationCandidate.birthdate,
+            'external-id': certificationCandidate.externalId,
+            'extra-time-percentage': certificationCandidate.extraTimePercentage,
+            'is-linked': !_.isNil(certificationCandidate.userId),
+          },
```

Comment:
```suggestion
            'extra-time-percentage': certificationCandidate.extraTimePercentage,
            'is-linked': !_.isNil(certificationCandidate.userId),
            'schooling-registration-id': certificationCandidate.schoolingRegistrationId,
          },
```

+ modifier la ligne 189 => ``expect(deserializedCertificationCandidate.schoolingRegistrationId).to.equal(certificationCandidate.schoolingRegistrationId);``